### ISSUE TITLE
docs for radians function

### ISSUE THREAD

**Issue by Ahrengot** [elm-lang/elm-lang.org#713]
http://package.elm-lang.org/packages/elm-lang/core/latest/Basics#radians

Looking at the source, it looks like `radians` is a function that just return whatever float you give it https://github.com/elm-lang/core/blob/14c737397c76ce1ea16c15009943ad56fa9e4b6d/src/Basics.elm#L67

I think this is more of a question than a bug, but can someone elaborate on purpose of this? When would you need something like this? In any case I think it would be helpful to add another line or two in the docs that explain what this is for, because when I stumpled on the description I sure was confused :)

> `radians : Float -> Float`
> Convert radians to standard Elm angles (radians).

Sorry if I'm just being obtuse here. That's really not the goal.

**Comment by ybakos** [comment]
@Ahrengot I agree. Shouldn't `radians` convert from degrees to radians, and furthermore, `degrees` should convert from radians to `degrees` ?

There's a semantics issue here. `degrees` should convert from radians to degrees, but it does the opposite.

**Comment by Gabgab2003** [comment]
I would say its for better self documentation, you could also see them as type annotations, so you have type degrees and type radians
`angle = radians 3.14` is more obvious than `angle = 3.14`

**Comment by ybakos** [comment]
@Gabgab2003 Thanks. Created a PR for the documentation on this.

**Comment by Gabgab2003** [comment]
@ybakos Thank you, hope it gets merged. I like the explanation in the PR :)
